installing/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -119,4 +119,30 @@ If no data appears in the ACS Console:
 * Ensure that at least one secured cluster is connected to your {product-title-managed-short} instance. For more information, see the "Installing secured cluster resources on each cluster" section.
 * Examine your Sensor pod logs to ensure that the connection to your {product-title-managed-short} instance is successful.
 * In the {ocp-short} cluster, navigate to *Platform Configuration* -> *Clusters* to verify that the components are healthy and view additional operational information.
-* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
\ No newline at end of file
+* Examine the values in the `SecuredCluster` API in the Operator on your local cluster to ensure that the *Central API Endpoint* has been entered correctly. This value should be the same value as shown in the *ACS instance* details in the {cloud-console}.
+
+[id="default-access-acs-console"]
+== Default access to ACS Console
+
+By default, the only authentication mechanism available to users is authentication by using Red Hat Single Sign-On (SSO).
+Red Hat SSO auth provider cannot be deleted or changed.
+However, you can change the minimum access role and add additional rules.
+
+[NOTE]
+====
+In case you want more control over access to your ACS console, it is recommended not to rely on default auth provider.
+You can set up your own IdP provider right after provisioning using one of the guides under `Operating -> Managing user access`
+====
+
+The default authentication provider has the required attribute `rh_org_id` set to the organization ID assigned to your account.
```

Comment:
@ivan-degtiarenko, could you please expand this section a bit with the info on how we integrate with RHSSO? Specifically:
* Create a dedicated dynamic client (in OIDC terms) per each cloud Central
* Forward some claims from RHSSO token to the ACS issued token, for example `rh_org_id` and `is_org_admin` for subsequent checks
* Re-use the same realm for all cloud Centrals, ensuring access control by verifying the org id claim in the instance; here it would be nice to link to the section where we recommend bringing their own IdP
* Set up some mappings to bootstrap access to the instance